Casbin/Enforcer.cs

```diff
@@ -247,7 +249,7 @@ public async Task<bool> EnforceAsync<TRequest>(EnforceContext context, TRequest
                         HandleExpressionResult(in context, ref session, in requestValues, in policyValues, Effector);
                     }
 
-                    if (session.Determined)
+                    if (session.Determined && (!session.IsChainEffector || effectChain.EffectExpressionType != EffectExpressionType.PriorityAllOverride))
```

Comment:
Is this necessary?

Reply:
Yes, without it, the test ```TestPriorityExplicitDenyOverrideModel``` will fail.

Reply:
What is the specific case of the failure? We have not changed the code about the test `TestPriorityExplicitDenyOverrideModel ` at this PR.

Reply:
> What is the specific case of the failure? We have not changed the code about the test `TestPriorityExplicitDenyOverrideModel ` at this PR.

You are right, the "error" is because I removed the whole branch when I test it :sweat_smile:. I restored it and everything is ok now.